bc_obps/registration/models.py

```diff
@@ -311,6 +311,10 @@ class Meta:
             models.Index(fields=["business_role"], name="contact_role_idx"),
         ]
 
+    def __str__(self) -> str:
+        fields = [f"{field.name}={getattr(self, field.name)}" for field in self._meta.fields]
+        return ' - '.join(fields)
+
```

Comment:
What is this function for? It looks like it's setting a set of fields to be in the format field-field-field? Just a question for my general knowledge :)

Reply:
It is helpful for debugging purposes (until I get around to setting up proper Python debugging in VS Code). It's so that when a dev throws in a `print(operation)` somewhere for debugging purposes, what gets printed to the python server console is the string representation of the `operation` object, rather than a very not-helpful `Operation <object>` (I forget exactly how it looks in the console, but basically you can't interrogate any of the values of the object through print statements)

Reply:
example output for an operation would be something like:

`id - 1 name - Operation 1 <other fields...>`